tests/integration.rs

```diff
@@ -6173,10 +6173,13 @@ mod parallel {
     }
 
     #[ignore]
```

Comment:
I'd suggest you remove the ignore and add `#[cfg(target_arch = "x86_64")]` so that it'd run only on aarch64. When it'll be fixed on x86, we can remove this `cfg` condition.

Reply:
Sounds good to me :) Thanks!

Reply:
Done